lua/nvcommunity/motion/hop/init.lua

```diff
@@ -4,6 +4,23 @@ local spec = {
     "smoka7/hop.nvim",
     cmd = { "HopWord", "HopLine", "HopLineStart", "HopWordCurrentLine" },
     opts = { keys = "etovxqpdygfblzhckisuran" },
+    init = function()
+      local ext = require('custom.chadrc').ui.extended_integrations or {}
+      table.insert(ext, "hop")
+      require('custom.chadrc').ui.extended_integrations = ext
+      require('base46').load_all_highlights()
```

Comment:
These are too many hooks for just one of this
Probably `au User LazyDone require('base46')` ...